### PR TITLE
Refactor: use eslint-cong-icij instead of project eslint config

### PR DESCRIPTION
**Pull request by marieglr**
This PR removes the local config of eslint and replaces it with the one from: https://github.com/ICIJ/eslint-config-icij.

The purpose is to have homogenous code style accross vue.js projects.

Mains config changes are:
- removing .eslintrc file and eslint related plugins and packages
- adding an eslinConfig property in package.json simply extending the eslint-config-icij package

Main fixes in this PR are: 
- removing the file extensions in file imports, especially `.vue`and `.js`extensions
- reordering imports for external imports to be on top (libs..) and project imports below, separated by a line break
- one occurrence of nextTick used without a callback
- one occurence of a computed property used in a data property ---> did not manage to fix this one, you

### CODE REVIEW

File: src/components/NamedEntityInContext.vue
Comment (caro3801) [inline src/components/NamedEntityInContext.vue:87]: this.scrollToFirstMark is not async

File: src/components/filter/FilterBoilerplate.vue
Comment (marieglr) [inline src/components/filter/FilterBoilerplate.vue:233]: `hasValues` here is a computed property, I tried to move the whole `collapseItems` to ba a computer property but it breaks the implem. Do you have a better suggestion?

File: src/main.js
Comment (caro3801) [inline src/main.js:5]: Todo now or later?

File: src/components/filter/FilterBoilerplate.vue
Comment (caro3801) [reply src/components/filter/FilterBoilerplate.vue:233]: It seems that this is something coming from vue 1.x (https://v2.vuejs.org/v2/guide/migration.html#cache-false-deprecated). The migration guide recommands to transform it as a method and call it like a method
```
     L9: 'filter--has-values': hasValues()
     L233: collapseItems: this.collapsedIfNoValues && !this.hasValues(),
```

from:
```
computed:{
     hasValues: {
        cache: false,
        get() {
          return this.$store.getters['search/hasFilterValues'](this.filter.name)
        }
    }
}
```
to
```
methods:{
     hasValues(): {
          return this.$store.getters['search/hasFilterValues'](this.filter.name)
    }
}
```

**Review by caro3801 — CHANGES_REQUESTED** [review]

File: src/main.js
Comment (marieglr) [reply src/main.js:5]: I will do it following this yes, but it needs for a new version of the package to be released. 

Do you think this single comment is a reason to block the merge of this PR with 'request changes' ? NB: you can stack your comments in a pull request review instead of making a new review for each comment. You can also comment without approving nor request changes if you don't want to approve but don't need for the next round of review to necessarily be made by you.

File: package.json
Comment (caro3801) [inline package.json:87]: this a test

File: src/components/filter/FilterBoilerplate.vue
Comment (caro3801) [inline src/components/filter/FilterBoilerplate.vue:522]: big problem delete please

**Review by caro3801 — APPROVED** [review]

**Pull request merged**
